### PR TITLE
[MPLUGIN-499] Add deprecate description in parameters table

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

**Review by michael-o — CHANGES_REQUESTED** [review]
The full stop after "Deprecated" is on purpose because `javadoc(1)` does the same: See: https://codehaus-plexus.github.io/plexus-utils/apidocs/org/codehaus/plexus/util/cli/Commandline.html#createArgument(). We wanted to resemblance.

**Comment by slawekjaranowski** [comment]
ok, next version

**Comment by michael-o** [comment]
Checking...

**Comment by michael-o** [comment]
Looks better now. I see one difference:
Before:
grafik

After:
grafik

The full stop is gone. Is that on purpose?

**Comment by slawekjaranowski** [comment]
Missing dot on list of goals is by accident. The same resources is used on goal details page:

image

It was:

```
Deprecated.:
```

**Comment by slawekjaranowski** [comment]
But it is a next issue on list of goals - plugin-info.htm when goal is deprecated we don't have a description.

**Comment by michael-o** [comment]
> Missing dot on list of goals is by accident. The same resources is used on goal details page:
> 
> image
> 
> It was:
> 
> ```
> Deprecated.:
> ```

Yes, one fixed the other one broken :-D Can you address?

**Comment by michael-o** [comment]
> But it is a next issue on list of goals - plugin-info.htm when goal is deprecated we don't have a description.

This isn't right. Deprecation should be first, line break, description.

**Comment by slawekjaranowski** [comment]
Now it will be as:
image

**Comment by michael-o** [comment]
> Now it will be as: image

Good. There should be a space between "Deprecated." and the actual text.

Let me test meanwhile.

File: maven-plugin-report-plugin/src/main/java/org/apache/maven/plugin/plugin/report/PluginOverviewRenderer.java [hunk 1]
```diff
@@ -144,15 +145,26 @@ private void renderGoalsSection(boolean hasMavenReport) {
...
+            sink.tableCell();
             if (StringUtils.isNotEmpty(mojo.getDeprecated())) {
-                description = "<strong>" + getI18nString("goal.deprecated") + "</strong> " + mojo.getDeprecated();
-            } else if (StringUtils.isNotEmpty(mojo.getDescription())) {
+                sink.division();
+                sink.inline(SinkEventAttributeSet.Semantics.STRONG);
+                sink.text(getI18nString("goal.deprecated"));
+                sink.text(".");
+                sink.inline_();
+                sink.rawText(mojo.getDeprecated());
```
Comment (michael-o) [inline maven-plugin-report-plugin/src/main/java/org/apache/maven/plugin/plugin/report/PluginOverviewRenderer.java:155]: Should be preceded with a space.

**Review by michael-o — CHANGES_REQUESTED** [review]

File: maven-plugin-report-plugin/src/main/java/org/apache/maven/plugin/plugin/report/PluginOverviewRenderer.java (hunk 1 above)
Comment (slawekjaranowski) [reply maven-plugin-report-plugin/src/main/java/org/apache/maven/plugin/plugin/report/PluginOverviewRenderer.java:156]: done

**Review by michael-o — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #805

**Pull request merged**